lib/fathead/mdn_css/parse.pl

```diff
@@ -297,6 +298,7 @@ sub create_abstract {
     my ( $description, $code, $initial_value ) = @_;
     if ($description) {
         $description = trim($description);
+        $description = escape_html($description) if $description =~ /</;
```

Comment:
@hchienjo this approach won't work. Internally we first decode all the HTML, before re-encoding the `<pre><code>` blocks.

You'll need to wrap any html elements that are meant to be escaped in `<code>` so the browser doesn't render it